Woof/Woof/View/SitterListView/SitterCardView/GeneralInfoSectionView.swift

```diff
@@ -15,6 +15,7 @@ struct GeneralInfoSectionView: View {
             VStack(alignment: .leading) {
                 Text(fullName)
                     .bold()
+                    .lineLimit(1)
                 Group {
                     if rating == 0 {
```

Comment:
I have been wondering about this question throughout the my part of similar code: shouldn't the view model pass data for rendering the number of rating stars? The only thing I couldn't figure out is how to implement it.

Reply:
Ok thanks!